app/src/components/buttons/HelpButtonTooltip.tsx

```diff
@@ -16,28 +16,52 @@ interface HelpButtonTooltipProps {
  * @return {*}
  */
 
-const HelpButtonTooltip = ({ content, children }: HelpButtonTooltipProps) => {
+const HelpButtonTooltip = ({ content, children }: HelpButtonTooltipProps): any => {
```

Comment:
We should avoid the "any" here.

(If this helps, there is a "PropsWithChildren<T>" type you can use, to add children to any existing type.